File: server-scala-play-server/app/model/ApplicationShortResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ApplicationShortResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ApplicationShortResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  applicationId: Option[Long],
  appKey: Option[String],
  title: Option[String],
  currentBuildVersion: Option[String],
  currentApiVersion: Option[String],
  appIcon: Option[AssetShortResponse],
  appLogo: Option[AssetShortResponse],
  active: Option[Boolean],
  about: Option[String],
  landingPageUrl: Option[String],
  landingPageFullUrl: Option[String],
  defaultImageFullUrl: Option[String],
  activityDescription: Option[String],
  platforms: Option[List[PlatformResponse]],
  categories: Option[List[CategoryResponse]],
  placementCount: Option[Int],
  billable: Option[BillableEntityShortResponse],
  hasAds: Option[Boolean],
  publicNotifications: Option[Boolean],
  urlScheme: Option[String],
  global: Option[Boolean],
  downloadUrls: Option[String],
  localAdsEnabled: Option[Boolean],
  trilatSettings: Option[TrilatAppSettings],
  returning: Option[String]
)

object ApplicationShortResponse {
  implicit lazy val applicationShortResponseJsonFormat: Format[ApplicationShortResponse] = Json.format[ApplicationShortResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ApplicationShortResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ApplicationShortResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long ApplicationId { get; set; }
    public string AppKey { get; set; }
    public string Title { get; set; }
    public string CurrentBuildVersion { get; set; }
    public string CurrentApiVersion { get; set; }
    public AssetShortResponse AppIcon { get; set; }
    public AssetShortResponse AppLogo { get; set; }
    public bool Active { get; set; }
    public string About { get; set; }
    public string LandingPageUrl { get; set; }
    public string LandingPageFullUrl { get; set; }
    public string DefaultImageFullUrl { get; set; }
    public string ActivityDescription { get; set; }
    public List<PlatformResponse> Platforms { get; set; }
    public List<CategoryResponse> Categories { get; set; }
    public int PlacementCount { get; set; }
    public BillableEntityShortResponse Billable { get; set; }
    public bool HasAds { get; set; }
    public bool PublicNotifications { get; set; }
    public string UrlScheme { get; set; }
    public bool Global { get; set; }
    public string DownloadUrls { get; set; }
    public bool LocalAdsEnabled { get; set; }
    public TrilatAppSettings TrilatSettings { get; set; }
    public string Returning { get; set; }
}


